eisbuk-admin/src/AppContent.tsx

```diff
@@ -30,8 +30,8 @@ import { getMonthStr } from "@/utils/helpers";
 
 import { ORGANIZATION } from "@/config/envInfo";
 
-import { calendarDaySelector } from "@/store/selectors";
-import { queryUserAdminStatus } from "@/store/actions/actions";
+import { queryUserAdminStatus } from "@/store/actions/authOperations";
+import { getCalendarDay } from "./store/selectors/app";
```

Comment:
Is it necessary for this to be `./` while the above is `@/` or is it a typo?

Reply:
Typo, many thanks, I hate those!